src/main/js/actions/grades.js

```diff
@@ -0,0 +1,14 @@
+import {
+  ADD_COURSEGPA,
+  SELECT_SORTBYGPA,
+} from './types';
+
+export const addGrades = gpa => ({
+  type: ADD_COURSEGPA,
+  gpa,
```

Comment:
I'm using the [flux standard action](https://github.com/redux-utilities/flux-standard-action) format, so data goes into a `payload` key. And it should also set an `error` key if the specific action can fail( which this one should be able to since the get request can fail). Look at `endGenerating` and how it is used for an example.

Reply:
Done.